.circleci/config.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           POSTGRES_USER: root
     parallelism: &parallelism 3
     steps: &steps
+      - run: skip-ci.sh && circleci step halt || true # skip Circle for guides
```

Comment:
Is it possible to have the script inline without the need to add another script file? I personally prefer a less readable line than a new file in the project that is used in a single point.

Reply:
A good place for the script could be `bin/circleci-halt-for-doc-only-changes` or `.circleci/halt-for-doc-only-changes` and move everything inside it. That way what it does should be self-evident and well contained.

Reply:
I like that suggestion @elia do you agree with this @kennyadsl?

Reply:
Yeah, `.circleci/bin/halt-for-doc-only-changes.sh` would be my preference